pyControl4/lock.py

```diff
@@ -0,0 +1,86 @@
+class C4Lock:
+    def __init__(self, C4Director, item_id):
+        """Creates a Control4 Lock object.
+
+        Parameters:
+            `C4Director` - A `pyControl4Ex.director.C4Director` object that
+            corresponds to the Control4 Director that the Lock is connected to.
+
+            `item_id` - The Control4 item ID of the Lock.
+        """
+        self.director = C4Director
+        self.item_id = item_id
+
+    async def is_locked(self):
+        """Returns True if it is currently Locked or False if it is Unlocked
+
+        Notes:
+            Relay State 0 == Locked
+            Relay State 1 == Unlocked
```

Comment:
I assume these refer to the raw variable value returned by Control4? If so, could you add a note like `Control4 variable values:` to this comment?